sklearn/compose/tests/test_column_transformer.py

```diff
@@ -2130,3 +2113,15 @@ def test_transformers_with_pandas_out_but_not_feature_names_out(
     ct.set_params(verbose_feature_names_out=False)
     X_trans_df1 = ct.fit_transform(X_df)
     assert_array_equal(X_trans_df1.columns, expected_non_verbose_names)
+
+
+def test_column_transformer_param_validation():
+    """Run the param validation check for `ColumnTransformer`."""
+    column_transformer = ColumnTransformer(
+        transformers=[
+            ("num", StandardScaler(), [0, 1]),
+            ("cat", OneHotEncoder(), [2, 3]),
+        ]
+    )
+
+    check_param_validation("ColumnTransformer", column_transformer)
```

Comment:
same remark as for Pipeline. What do you think about extending the common test to check these ones as well ?